example/src/main/java/com/github/appintro/example/ui/default/DefaultIntro2.kt

```diff
@@ -53,6 +54,8 @@ class DefaultIntro2 : AppIntro2() {
                 backgroundDrawable = R.drawable.back_slide5
         ))
 
+        setNextArrowColor(Color.RED)
+        setSkipArrowColor(Color.BLUE)
```

Comment:
Can you move those two to either `CustomBackgroundIntro.kt`?
The `DefaultIntro2` is used for screenshots in the README and we would rather keep it 
less customized.